prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -116,34 +125,71 @@ public static FeedSLAMonitoringService get() {
 
     @Override
     public void onAdd(Entity entity) throws FalconException {
+        Set<String> currentClusters = DeploymentUtil.getCurrentClusters();
         if (entity.getEntityType() == EntityType.FEED) {
             Feed feed = (Feed) entity;
             // currently sla service is enabled only for fileSystemStorage
             if (feed.getLocations() != null) {
-                Set<String> currentClusters = DeploymentUtil.getCurrentClusters();
                 for (Cluster cluster : feed.getClusters().getClusters()) {
                     if (currentClusters.contains(cluster.getName())) {
```

Comment:
Adding one entry per cluster will cause multiple entries for the same feed in the table when a feed has multiple clusters.